src/components/MarketActions/Borrow/MarketLeverageBorrow.tsx

```diff
@@ -404,3 +406,35 @@ export default function MarketLeverageBorrow({
     </>
   );
 }
+
+function getTrackingPayload(market: MarketNonIdle, action: MarketLeveragedBorrowAction | null, tag: string) {
+  const basePayload = {
+    tag,
+    marketId: market.marketId,
+  };
+
+  if (!action || action.status !== "success") {
+    return basePayload;
+  }
+
+  const collateralDelta = descaleBigIntToNumber(
+    action.positionCollateralChange.after - action.positionCollateralChange.before,
+    market.collateralAsset.decimals
+  );
+  const loanDelta = descaleBigIntToNumber(
+    action.positionLoanChange.after - action.positionLoanChange.before,
+    market.loanAsset.decimals
+  );
+  const collateralDeltaUsd = market.collateralAsset.priceUsd
+    ? collateralDelta * market.collateralAsset.priceUsd
+    : undefined;
+  const loanDeltaUsd = market.loanAsset.priceUsd ? loanDelta * market.loanAsset.priceUsd : undefined;
+
+  return {
+    ...basePayload,
+    collateralAmount: Math.abs(collateralDelta),
+    loanAmount: Math.abs(loanDelta),
+    collateralAmountUsd: collateralDeltaUsd ? Math.abs(collateralDeltaUsd) : "",
```

Comment:
Bug here, in the actions it's possible `loanAmount` or `collateralAmount` is 0, which causes us to put "" instead of 0 here. Would expect "" ONLY when the priceUsd is not found.

Also, is "" the best thing to put here (from querying perspective later)?

Reply:
Example:

<img width="615" alt="Screenshot 2025-06-06 at 15 24 49" src="https://github.com/user-attachments/assets/7f89abf6-1b1b-4a3a-96a1-290a3cd8530d" />

Reply:
Im good with leaving "", just will need to know for when we post-process.